### PR TITLE
add 0.12.0 experimental release

### PR DESCRIPTION
**Pull request by gandrewstone**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,10 +12,10 @@ The following will install the project and start the server, listening on port 8
 npm install grunt-cli -g
 npm install bower -g
 npm install forever -g
-git clone git@github.com:gandrewstone/BitcoinUnlimitedWeb.git
+git clone git@github.com:BitcoinUnlimited/BitcoinUnlimitedWeb.git
```
Comment (thofmann) [inline README.md]: :+1:

File: src/public/views/components/download/experimental-releases.jsx [hunk 2]
```diff
@@ -16,16 +16,18 @@ export default React.createClass({
...
+                    <p>32-bit: Version: 0.12.0, Feb 24, 2015: <a href='/downloads/bitcoinUnlimited-0.12.0-win32-setup.exe'> bitcoinUnlimited-0.12.0-win32-setup.exe</a> <a href='/downloads/bitcoinUnlimited-0.12.0-win32.zip'> (zip)</a></p>
                     <br />
                     <h4>Linux</h4>
-                    <p>Nothing right now</p>
+                    <p>64-bit: Version: 0.12.0, Feb 24, 2015:<a href='/downloads/bitcoinUnlimited-0.12.0-linux64.tar.gz'> bitcoinUnlimited-0.12.0-linux64.tar.gz</a></p>
+                    <p>32-bit: Version: 0.12.0, Feb 24, 2015: <a href='/downloads/bitcoinUnlimited-0.12.0-linux32.tar.gz'> bitcoinUnlimited-0.12.0-linux32.tar.gz</a></p>
                     <br />
                     <h4>Source Code</h4>
-                    <p>Github <a href='https://github.com/gandrewstone/BitcoinUnlimited'>here</a>.</p>
+                    <p>Github branch 0.12bu <a href='https://github.com/gandrewstone/BitcoinUnlimited'>here</a>.</p>
```
Comment (thofmann) [inline src/public/views/components/download/experimental-releases.jsx]: This link redirects to the BitcoinUnlimited/BitcoinUnlimited project, but we can update it to skip the redirect. I'll add an issue and try to fix that soon.

**Comment by thofmann** [comment]
You can ignore the Travis CI build failure. I need to configure the build process now that Travis has been enabled for the project.

File: src/public/views/components/download/official-release.jsx [hunk 3]
```diff
@@ -21,8 +21,8 @@ export default React.createClass({
                     <h3>Binary Downloads</h3>
                     <br />
                     <h4>Windows</h4>
-                    <p>64-bit: Version: 0.11.2, Dec 22, 2015: <a href='/downloads/bitcoinUnlimited-0.11.2-win64-setup.exe'> bitcoinUnlimited-0.11.2-win64-setup.exe</a> <a href='public/downloads/bitcoinUnlimited-0.11.2-win64.zip'> (zip)</a></p>
-                    <p>32-bit: Version: 0.11.2, Dec 22, 2015: <a href='/downloads/bitcoinUnlimited-0.11.2-win32-setup.exe'> bitcoinUnlimited-0.11.2-win32-setup.exe</a> <a href='public/downloads/bitcoinUnlimited-0.11.2-win32.zip'> (zip)</a></p>
+                    <p>64-bit: Version: 0.11.2, Dec 22, 2015: <a href='/downloads/bitcoinUnlimited-0.11.2-win64-setup.exe'> bitcoinUnlimited-0.11.2-win64-setup.exe</a> <a href='/downloads/bitcoinUnlimited-0.11.2-win64.zip'> (zip)</a></p>
```
Comment (thofmann) [inline src/public/views/components/download/official-release.jsx]: :+1:

Good catch!

**Comment by thofmann** [comment]
utACK

File: src/public/views/components/download/experimental-releases.jsx (hunk 2 above)
Comment (gandrewstone) [inline src/public/views/components/download/experimental-releases.jsx]: fixed

**Pull request merged**